source/ranges.tex

```diff
@@ -3847,6 +3850,9 @@
 \end{itemize}
 
 \pnum
+The member \grammarterm{typedef-name} \tcode{iterator_category} is defined
+if and only if \tcode{Base} models \tcode{forward_range}.
```

Comment:
\tcode{forward_range} -> \libconcept{forward_range}

Reply:
That's already fixed.

Reply:
I don't see the fix.  just updated, and it's still:
`if and only if \exposid{Base} models \tcode{forward_range}.`

Reply:
Ah, nice catch.

Reply:
I have a fix - was about to push. shall I or do you want to?

Reply:
I don't have push permission. I'm not even a committee member.

Reply:
sorry, thought you were Jens :).  I've pushed.
BTW, many thanks for your reviews! *Really* appreciated!!! Would love to have you in the committee!

Reply:
I hope to join you someday.